### DISCUSSION TITLE
ChunkManager.configure

### DISCUSSION THREAD

**Discussion by abhilash-u** [callstack/repack#100] [Q&A] [answered]
Hi Repack Team,

In the TesterApp, the ChunkManager, resolveRemoteChunk url is configured to port 5000.
Whereas the webpack devServer is running on port 8081.

Firstly, How is then the TesterApp able to load the chucks from port 8081, eventhough the ChunkManager is pointing to port 5000.
Secondly, what would it take to make webpack dev server run on other than 8081 port.
Thank you.

**Comment by zamotany** [comment, marked as answer]
In development mode (`__DEV__ === true`) `ChunkManager` does not use `resolveRemoteChunk` unless `forceRemoteChunkResolution` is set to `true`. That's why it's able to load chunks even though it points to port `5000`.
More: https://re-pack.netlify.app/docs/api/react-native/classes/ChunkManager

In development mode, there's no need to change the port, but if you want to, it's the same process as changing the port with Metro (`--port <port>` flag and then you need to tell native code to use different port).
